_includes/CheatSheet.md

```diff
@@ -1,213 +1,214 @@
-An existing instance is always valid. Instead of returning null or invalid values, exceptions are thrown.
+| method name or prefix | usage                                        | examples                                                        |
+| --------------------- | -------------------------------------------- | --------------------------------------------------------------- |
+| `.of`                 | static factory method for building by parts  | `LocalDate.of(2016, 2, 23)` <br> `LocalDate.ofInstant(i)`       |
+| `.parse`              | static factory method for parsing strings    | `LocalDate.parse('2016-02-23')` <br> `LocalTime.parse('12:34')` |
+| `.is`                 | checks for certain conditions                | `t1.isAfter(t2)` <br> `d1.isLeapYear()`                         |
+| `.equals`             | checks for equivalence between two instances | `t1.equals(t2)`                                                 |
+| `.with`               | the immutable equivalent of a setter         | `d.withDayOfMonth(1)` <br> `t.withHour(9)`                      |
+| `.plus`               | adds an amount to an object                  | `t.plusMinutes(5)` <br> `d.plus(3, ChronoUnit.YEARS)`           |
+| `.minus`              | subtracts an amount from an object           | `t.minusHours(1)` <br> `d.minus(1, ChronoUnit.DAYS)`            |
+| `.to`                 | converts this object to another type         | `dt.toLocalDate()` <br> `d1.until(d2).toTotalMonths()`          |
+| `.at`                 | combines one object with another             | `date.atTime(time)` <br> `localDate.atZone(tz)`                 |
 
-## LocalDate
+Note that getter methods for instance properties omit the get keyword: `d.year()`, not ~~`d.getYear()`~~.
+
+### Try it out
+
+**Tip:** You can run the examples on this page in your browser console. All `js-joda` classes are imported into the global namespace of this web page.
 
-A LocalDate represents a date without a time and time-zone in the ISO-8601 calendar system, such as 2007-12-24.
+![image](https://user-images.githubusercontent.com/2136620/47266364-9be10b00-d535-11e8-9a38-b53a6c16728d.png)
 
-### Create a LocalDate
+## LocalDate
+
+A `LocalDate` represents a date with **no time** and **no time zone** in the [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) calendar system, such as 2007-12-24.
 
-```javascript
+### Create a `LocalDate`
 
-// obtain the current date in the system default timezone, e.g. 2016-02-23
+```JavaScript
+// obtain the current date in the system default time zone, e.g. 2016-02-23
 LocalDate.now();
 
-// obtain the current date in the utc timezone, e.g. 2016-02-23
+// obtain the current date in the UTC time zone, e.g. 2016-02-23
 LocalDate.now(ZoneOffset.UTC);
 
-// obtain an instance of LocalDate from an ISO8601 formatted text string
-LocalDate.parse('2016-02-23');
+// obtain an instance of LocalDate from an ISO 8601 formatted text string
+LocalDate.parse("2016-02-23");
```

Comment:
this is nitpicking... but we prefer single over double quotes in our eslint config https://github.com/js-joda/eslint-config-js-joda/blob/master/.eslintrc.js#L18  so i *think* it makes sense to use single quotes here as well... this is merely a matter of taste i think :) 
```suggestion
LocalDate.parse('2016-02-23');
```